dev-aws/kafka-shared/iam/iam.tf

```diff
@@ -1,267 +0,0 @@
-resource "kafka_topic" "iam_cerbos_audit_v1" {
```

Comment:
Why are you leaving empty files after removing contents?

Reply:
I'll delete the whole module after the applier will delete the resources